Cargo.toml

```diff
@@ -252,7 +253,8 @@ lazy_static = { version="1.3" }
 textwrap = { version="0.14", features=["terminal_size"] }
 uucore = { version=">=0.0.11", package="uucore", path="src/uucore" }
 selinux = { version="0.2", optional = true }
-ureq = "2.4.0"
+reqwest = { version="0.11.9", features = ["blocking"] }
```

Comment:
this is pulling a lot of new deps, is it possible to avoid that?

Reply:
I'm unaware of any way to reduce the dependencies, besides simply not using `reqwest`.

We could simply wait for `ring` to add RISC-V support, and use the existing `ureq` crate. There's already a PR for this: https://github.com/briansmith/ring/pull/1459

Reply:
So you're suggesting that we close this PR? We could create an issue referencing that PR instead.

Reply:
We could close this PR and create a different one _if we want to take that route._
The downside is that we would still be depending on `ring`'s architecture support, which is problematic for other systems (see #3216).

I would prefer to try and resolve the CI issues with openssl instead.